lgtm.yml

```diff
@@ -0,0 +1,10 @@
+extraction:
```

Comment:
This could be a hidden file `.lgtm.yml` if you'd prefer. Those seem to be the only two filenames allowed.

Reply:
The hidden one please. I did try something like this a few weeks ago but my Debian-foo wasn't strong enough to get over the pip warning you'll probably get. Thanks for looking at this!